static/generated/github-contributors.json

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
Looks like you accidentally included this auto generated file.

Reply:
Thanks for catching my mistake! I'll correct that.

Reply:
I've deleted the file. Would you mind giving my PR another look?

Reply:
Usually, such things should be fixed with `git commit --amend` instead of adding as a new commit because we do not want to have it on the master as it is only related to this PR.
These docs might help https://zulip.readthedocs.io/en/latest/git/fixing-commits.html, https://zulip.readthedocs.io/en/latest/git/using.html#force-push-changes-to-github-after-you-ve-altered-your-history

> Would you mind giving my PR another look?

I'm not really familiar with other parts of PR, was having a quick look and found that to tell.